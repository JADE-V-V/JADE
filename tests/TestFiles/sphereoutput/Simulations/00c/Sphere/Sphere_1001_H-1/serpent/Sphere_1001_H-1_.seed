1710424223710
