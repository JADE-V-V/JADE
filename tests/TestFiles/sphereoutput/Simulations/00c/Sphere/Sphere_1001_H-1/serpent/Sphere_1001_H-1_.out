
 --- Table  1: Summary of nuclide data: 

 Data for 2 nuclides included in calculation:

 - 1 Transport nuclides
 - No dosimetry nuclides
 - No decay nuclides
 - 6 reaction channels
 - 4 special reactions
 - No transmutation reactions
 - No transmutation branch reactions
 - No decay reactions
 - No decay branch reactions
 - All reaction linked to target

 Columns:

  1. Index
  2. Level (first occurence in transmutation paths)
  3. Name (nuclide identifier data)
  4. Library ID
  5. Primary data type: TRA = transport
                        DOS = dosimetry
                        SAB = thermal scattering
                        DEC = only decay data
                        STR = only structural data (mass)
                        PHO = photon interaction data
  6. Z (atomic number)
  7. A (isotope number)
  8. I (isomeric state)
  9. Atomic weight
 10. Temperature (in Kelvin)
 11. Decay constant (1/s)
 12. Flags: I - nuclide is present in initial composition
            B - nuclide is involved in burnup calculation
            A - nuclide is produced as activation product
            D - nuclide is produced as decay product
            F - nuclide is produced as fission product
            B - nuclide is produced in decay or transmutation branching
            T - nuclide has neutron transport cross section data
            S - nuclide has S(a,b) data
            I - nuclide has isomeric branching reactions
            F - nuclide is fissile
            P - nuclide is a delayed neutron precursor
            D - nuclide has delayed neutron data
            U - nuclide has uses ures ptable data
            T - nuclide cross sections are adjusted by TMS
            D - nuclide has dosimetry cross section data
            R - nuclide has radioactive decay data
            N - nuclide has neutron-induced fission yield data
            S - nuclide has spontaneous fission yield data
            P - nuclide has photon transport cross section data
            T - nuclide has supplementary transmutation cross section data
            D - nuclide is used to provide 0K data for DBRC

    1   0  1001.00c      00c  TRA    1   1 0    1.00782   294.0          N/A  I-----T--------------  hydrogen (H-1)
    2   0  1000.84p      84p  PHO    1   0 0    1.00798     0.0          N/A  I-----------------P--  natural hydrogen (H-nat)

 --- Table  2: Reaction and decay data: 

 Nuclide    1 / 2 : 1001.00c -- hydrogen (H-1)

 Pointers                     : 5078 334826
 Primary type                 : Transport
 Nuclide ZAI                  : 10010
 Library ID                   : 00c
 Cross section data file      : Lib80x/Lib80x/H/1001.800nc
 Atomic weight (AW)           : 1.00782
 Atomic weight ratio (AWR)    : 0.99917
 Temperature                  : 294.0 Kelvin
 XS temperature               : 294.0 Kelvin
 Minimum and maximum energy   : 1.00000E-11 2.00000E+01
 Half-life                    : N/A
 Specific ingestion toxicity  : N/A
 Specific inhalation toxicity : N/A

 - Nuclide is not involved in burnup calculation
 - Nuclide has neutron transport data
 - Nuclide has no photon transport data
 - Nuclide has no S(a,b) data
 - Nuclide has no isomeric branching data
 - Nuclide is not fissile
 - Nuclide is not a delayed neutron precursor
 - Nuclide has no delayed neutron data
 - Nuclide has no ures ptable data
 - Nuclide temperature is not adjusted with TMS
 - Nuclide has no dosimetry data
 - Nuclide has no decay data
 - Nuclide has no supplementary transmutation cross section data
 - Nuclide has no NFY data
 - Nuclide has no SFY data

 - 2 reaction channels
 - 3 special reactions
 - No transmutation reactions
 - No transmutation branch reactions
 - No decay reactions
 - No decay branch reactions
 - No transmutation paths

 2 reactions channel:

   1    MT =      2  Q =  0.00000E+00 MeV  Emin = 1.00000E-11 MeV  frac = 1.00000E+00  : elastic scattering
   2    MT =    102  Q =  2.22465E+00 MeV  Emin = 1.00000E-11 MeV  frac = 1.00000E+00  : (n,gamma)

 3 special reactions:

   1    MT =    202  Q =  0.00000E+00 MeV  Emin = 1.00000E-11 MeV  : total photon production
   2    MT =    204  Q =  0.00000E+00 MeV  Emin = 1.00000E-11 MeV  : total deuteron production
   3    MT =    444  Q =  0.00000E+00 MeV  Emin = 1.00000E-11 MeV  : damage-energy production

 Nuclide    2 / 2 : 1000.84p -- natural hydrogen (H-nat)

 Pointers                     : 5608 299130
 Primary type                 : Photon interaction
 Nuclide ZAI                  : 10000
 Library ID                   : 84p
 Cross section data file      : xdata/mcplib84
 Atomic weight (AW)           : 1.00798
 Atomic weight ratio (AWR)    : 0.99924
 Temperature                  : 0.0 Kelvin
 XS temperature               : 0.0 Kelvin
 Minimum and maximum energy   : 1.00000E-03 1.00000E+05
 Half-life                    : N/A
 Specific ingestion toxicity  : N/A
 Specific inhalation toxicity : N/A

 - Nuclide is not involved in burnup calculation
 - Nuclide has no neutron transport data
 - Nuclide has photon transport data
 - Nuclide has no S(a,b) data
 - Nuclide has no isomeric branching data
 - Nuclide is not fissile
 - Nuclide is not a delayed neutron precursor
 - Nuclide has no delayed neutron data
 - Nuclide has no ures ptable data
 - Nuclide temperature is not adjusted with TMS
 - Nuclide has no dosimetry data
 - Nuclide has no decay data
 - Nuclide has no supplementary transmutation cross section data
 - Nuclide has no NFY data
 - Nuclide has no SFY data

 - 4 reaction channels
 - 1 special reactions
 - No transmutation reactions
 - No transmutation branch reactions
 - No decay reactions
 - No decay branch reactions
 - No transmutation paths

 4 reactions channel:

   1    MT =    504  Q =  0.00000E+00 MeV  Emin = 1.00000E-03 MeV  : Compton scattering
   2    MT =    502  Q =  0.00000E+00 MeV  Emin = 1.00000E-03 MeV  : Rayleigh scattering
   3    MT =    522  Q =  0.00000E+00 MeV  Emin = 1.00000E-03 MeV  : photoelectric effect
   4    MT =    516  Q =  0.00000E+00 MeV  Emin = 1.00000E-03 MeV  : pair production

 1 special reaction:

   1    MT =    301  Q =  0.00000E+00 MeV  Emin = 1.00000E-03 MeV  : total heat production

 --- Table  1: Summary of material compositions: 

 1 materials included in calculation


Material "1":

 - Material is not burnable
 - Material is included in majorant
 - Material is included in geometry
 - Atom density 4.23050E-02 1/barn*cm
 - Mass density 7.08000E-02 g/cm3
 - Volume 0.00000E+00 cm3
 - Mass 0.00000E+00 g
 - Photon emission rate 0.00000E+00 1/s
 - Neutron emission rate 0.00000E+00 1/s
 - 1 nuclides in composition
 - No nuclides associated with S(a,b) data
 - 111.16 kb of memory allocated for data

Isotopic composition (non-zero densities):

-------------------------------------------------------------------
 Nuclide    a. weight   temp      a. dens      a. frac      m. frac
-------------------------------------------------------------------
  1001.00c    1.00782  294.0  4.23050E-02  1.00000E+00  1.00000E+00 
       sum                    4.23050E-02  1.00000E+00  1.00000E+00
-------------------------------------------------------------------
